lib/send_sms.rb

```diff
@@ -0,0 +1,33 @@
+require 'twilio-ruby'
+require 'dotenv'
+
+class Send_SMS
```

Comment:
Style/ClassAndModuleCamelCase: Use CamelCase for classes and modules. (https://github.com/bbatsov/ruby-style-guide#camelcase-classes)